index.html

```diff
@@ -17,12 +17,13 @@
           <a href="#" class="logo">My Logo</a>
           <i class="fa-solid fa-bars"></i>
         </nav>
-        <nav id="menu">
-          <ul>
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
+        <nav id="menu" class="menu-container">
+          <ul class="menuList">
+            <li><a href="#work-section">Portfolio</a></li>
+            <li><a href="#about-myself">About</a></li>
+            <li><a href="#contact-me">Contact</a></li>
           </ul>
+          <i class="fa-solid fa-xmark close-btn"></i>
```

Comment:
- [ ] it would be better if your menu shouldn't displayed in the desktop version and make sure that the header of the desktop version that should be displayed on desktop
- Yours 
<img width="1440" alt="Screenshot 2022-11-22 at 16 01 03" src="https://user-images.githubusercontent.com/61790587/203347991-785e0ea3-c7ad-4fea-881b-458dca065af3.png">